common/src/model_registration/register.py

```diff
@@ -65,7 +65,7 @@ def parse_args():
     parser.add_argument(
         "--model_metadata",
         type=str,
-        help="YAML file that contains model metadata confirming to Model V2",
+        help="Json file that contains model metadata confirming to Model V2",
```

Comment:
metadata file will be JSON?

Reply:
Yes, previously we forgot to update description

Reply:
```suggestion
        help="JSON file that contains model metadata confirming to Model V2",
```